src/process.js

```diff
@@ -17,7 +17,7 @@ export class Process {
     this.#proc.kill('SIGINT');
   }
 
-  #publish(eventName, eventData) {
+  #subscribe(eventName, eventData) {
```

Comment:
@tony-go changing the name for subscribe can be confusing. What this method does is not subscribe but publish or emit the event to the subscribers of the event.

Reply:
@tony-go , finally, with your suggested change, this method is no more used :D